### PR TITLE
Fw-4688, use hosted cognito login pages

### PR DESCRIPTION
**Pull request by sarahfirstvoices**
### Description of Changes
* temporarily added both "sign in" and "sign out" to the user menu for testing (will remove with fw-4636)
* log in and out via hosted cognito ui
* after login, user is redirected to their original page
* after logout, user is redirected to main page
* new environment vars are documented in the readme -- ask me if you need the values for local dev

### Checklist

- [x] README / documentation has been updated if needed
- [x] PR title / commit messages contain Jira ticket number if applicable
- [x] Tests have been updated / created if applicable

### Reviewers Should Do The Following:

- [x] Review the changes here
- [x] Pull the branch and test locally

### Additional Notes

N/A

### CODE REVIEW

File: src/components/App/AppContainer.js [hunk 1]
```diff
@@ -13,7 +13,6 @@ import ErrorHandler from 'components/ErrorHandler'
 import LandingPage from 'components/LandingPage'
 import Languages from 'components/Languages'
 import Loading from 'components/Loading'
-import Login from 'components/Login'
```
Comment (sarahfirstvoices) [inline src/components/App/AppContainer.js:16]: RIP components/Login 😢

File: src/components/AppNavBar/AppNavBarPresentation.js [hunk 2]
```diff
@@ -39,28 +39,44 @@ function AppNavBarPresentation({ isHome = false }) {
     if (mobileLandingNavbarOpen) {
       setMobileLandingNavbarOpen(false)
     }
-  }, [useLocation()])
+  }, [useLocation(), setMobileLandingNavbarOpen])
```
Comment (sarahfirstvoices) [inline src/components/AppNavBar/AppNavBarPresentation.js:42]: Requested by es-lint

File: src/components/UserMenu/UserMenuPresentation.js [hunk 3]
```diff
@@ -105,58 +102,62 @@ function UserMenuPresentation({ currentUser }) {
...
-                    <a href="/register?requestedUrl=/register">
-                      <div className={`${active ? menuItemActiveClass : menuItemInactiveClass} ${menuItemBaseClass}`}>
-                        Register
-                      </div>
-                    </a>
-                  )}
-                </Menu.Item> */}
-              </>
-            ) : (
+            {/* {currentUser?.username === 'Guest' || !currentUser?.username ? ( */}
```
Comment (sarahfirstvoices) [inline src/components/UserMenu/UserMenuPresentation.js:105]: This is the temporary change to always show both sign in and sign out

File: src/index.js [hunk 4]
```diff
@@ -29,12 +32,37 @@ const queryClient = new QueryClient({
...
+      // remove url params to complete the login
+      window.history.replaceState({}, document.title, window.location.pathname)
+      window.location.reload()
+    }
+  },
+}
+
+if (GlobalConfiguration.END_SESSION_URL) {
+  oidcConfig.metadataSeed = {
+    end_session_endpoint: GlobalConfiguration.END_SESSION_URL,
```
Comment (sarahfirstvoices) [inline src/index.js:56]: Cognito has a special logout url that isn't advertized in the metadata, so we have to configure it manually

**Review by sahil97 — APPROVED** [review]
LGTM !

**Pull request merged**